docs/3.0/resources.md

```diff
@@ -382,11 +382,13 @@ Resources have a few options available for customization.
 
 :::option `self.title`
 
-Initially, the `title` attribute is set to `:id`, so the model's `id` attribute will be used to display the resource in search results and belongs select fields. You usually change it to something more representative, like the model's `title`, `name` or `label` attributes.
+The resource possesses an intelligent method for retrieving the title. It attempts to invoke a series of record methods in the following order: `name`, `title`, `label`, and `id`.
```

Comment:
```suggestion
Each Avo resource will try to figure out what the title of a record is. It will try the following attributes in order `name`, `title`, `label`, and fallback to the `id`.
```